### ISSUE TITLE
Bug: Playground errors out due to "Accordion" component export

### ISSUE THREAD

**Issue by sylwiavargas** [wonderflow-bv/wanda#251]
### What happened?

**tl;dr The playground errors out because "@wonderflow/react-components"' has no exported member 'Accordion'.**

PS. I can't assign the project on the right but I clicked the checkbox at the bottom of this template - I'm sorry!

---
### 1. [Open the playground](https://design.wonderflow.ai/playground)
Once the WebContainers boot, you will see an empty preview:
<img width="1139" alt="Screen Shot 2022-09-29 at 10 52 03 AM" src="https://user-images.githubusercontent.com/45401242/193000734-b67b69aa-7d7d-4b3c-9925-840a3ed8721c.png">

### 2. Open the Dev Tools Console
You will see the following error:
<img width="301" alt="Screen Shot 2022-09-29 at 10 53 25 AM" src="https://user-images.githubusercontent.com/45401242/193001064-6ed5521c-0a9a-4a2d-9f3e-a810c432ead4.png">

### 3. Navigate to `/src/routes/accordion.tsx`
You will see the following error: `Module '"@wonderflow/react-components"' has no exported member 'Accordion'.`

<img width="1142" alt="Screen Shot 2022-09-29 at 10 55 24 AM" src="https://user-images.githubusercontent.com/45401242/193001494-06265f87-3853-4067-b3b6-db03e4f43374.png">

### 4. For the playground to work, I deleted line 7 in `/src/routes/index.ts` (`export { AccordionPage } from './accordion';
`)
<img width="1142" alt="Screen Shot 2022-09-29 at 10 57 07 AM" src="https://user-images.githubusercontent.com/45401242/193002022-1bfe2b5c-cf8c-4d67-b529-76a1b7df625a.png">

### Version

online playground :) 

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
Module '"@wonderflow/react-components"' has no exported member 'Accordion'.
```

### Code of Conduct

- [X] I've searched across open and closed issues for similiar bug reports

### Project assignment

- [X] I've assigned Wanda Design System project from the sidebar on the right.

**Comment by equinusocio** [comment]
Hi @sylwiavargas Thanks for the report. The issue should be fixed now 👍🏻
